core/src/test/java/com/datastax/oss/driver/internal/core/cql/QueryTraceFetcherTest.java

```diff
@@ -79,7 +79,7 @@ public class QueryTraceFetcherTest {
   @Mock private NettyOptions nettyOptions;
   @Mock private EventExecutorGroup adminEventExecutorGroup;
   @Mock private EventExecutor eventExecutor;
-  @Mock private InetAddress address;
+  private InetAddress address = InetAddress.getLoopbackAddress();
```

Comment:
Mockito can no longer mock `InetAddress` as it becomes `final` in newer JDK versions.